webpack.production.config.js

```diff
@@ -38,6 +38,11 @@ module.exports = {
     ]
   },
   plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify('production')
```

Comment:
Was copied and pasted from the webpack website. It is because the string needs to be like this:

`'NODE_ENV': '"production"'`